portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/AnalyticsCloudIndividualTests.testcase

```diff
@@ -27,6 +27,53 @@ definition {
 		}
 	}
 
+	@description = "Set custom range filter with data for individual activity"
+	@priority = "5"
+	test CanSetDataWithCustomRangeOnIndividualActivitiesCard {
+		Navigator.openURL();
+		ACUtils.addSiteAndPage();
+		ACUtils.createDMAndAddToPage();
+		ACDXPSettings.connectDXPtoAnalyticsCloud(siteName = "Site Name");
+
+		for (var n : list "1,2,3,4") {
+			Navigator.gotoSitePage(
+				pageName = "AC Page",
+				siteName = "Site Name");
+
+			DMNavigator.gotoDocumentCP(dmDocumentTitle = "DM AC Title");
+		}
+
+		Pause(locator1 = "10000");
```

Comment:
Hey @calvinctvo , could you please double-check if the test is able to pass without this `Pause(locator1 = "10000");`? 

I think we don't need to have it since we are only viewing an asset. When we were submitting/downloading an asset I think it is needed.

Reply:
Also, could you please check the same situation on the previous test case? `CanSetEmptyDataCustomRangeOnIndividualActivitiesCard`

Thanks!